examples/js/renderers/RaytracingRenderer.js

```diff
@@ -456,7 +458,13 @@ THREE.RaytracingRenderer = function ( parameters ) {
 				blockX = 0;
 				blockY += blockSize;
 
-				if ( blockY >= canvasHeight ) return;
+				if ( blockY >= canvasHeight ) {
+
+					dispatch( { type: "complete" } );
```

Comment:
Yet another tweak... Why not just doing `scope.dispatchEvent( { type: "complete" } )`?